### PR TITLE
Add nncf compression configs

### PR DESCRIPTION
**Pull request by LeonidBeynenson**

### CODE REVIEW

**Comment by LeonidBeynenson** [comment]
[log_009_2020-12-22_04-30-32.zip](https://github.com/openvinotoolkit/training_extensions/files/5727443/log_009_2020-12-22_04-30-32.zip)

File: pytorch_toolkit/object_detection/model_templates/face-detection/face-detection-0202/template.yaml [hunk 1]
```diff
@@ -43,7 +43,10 @@ output_format:
   openvino:
     default: true
     input_format: BGR
-compression: null
+optimisations:
+  nncf_quantization:
+    config: "compression_config.json"
```
Comment (evgeny-izutov) [inline pytorch_toolkit/object_detection/model_templates/face-detection/face-detection-0202/template.yaml:48]: Could you please remove string enclosing?

File: pytorch_toolkit/object_detection/model_templates/face-detection/face-detection-0207/template.yaml [hunk 2]
```diff
@@ -43,7 +43,10 @@ output_format:
   openvino:
     default: true
     input_format: BGR
-compression: null
+optimisations:
+  nncf_quantization:
+    config: "compression_config.json"
```
Comment (evgeny-izutov) [inline pytorch_toolkit/object_detection/model_templates/face-detection/face-detection-0207/template.yaml:48]: ditto

File: pytorch_toolkit/object_detection/model_templates/person-detection/person-detection-0200/template.yaml [hunk 3]
```diff
@@ -43,7 +43,10 @@ output_format:
   openvino:
     default: true
     input_format: BGR
-compression: null
+optimisations:
+  nncf_quantization:
+    config: "compression_config.json"
```
Comment (evgeny-izutov) [inline pytorch_toolkit/object_detection/model_templates/person-detection/person-detection-0200/template.yaml:48]: ditto

File: pytorch_toolkit/object_detection/model_templates/person-detection/person-detection-0201/template.yaml [hunk 4]
```diff
@@ -43,7 +43,10 @@ output_format:
   openvino:
     default: true
     input_format: BGR
-compression: null
+optimisations:
+  nncf_quantization:
+    config: "compression_config.json"
```
Comment (evgeny-izutov) [inline pytorch_toolkit/object_detection/model_templates/person-detection/person-detection-0201/template.yaml:48]: ditto

File: pytorch_toolkit/object_detection/model_templates/person-detection/person-detection-0202/template.yaml [hunk 5]
```diff
@@ -43,7 +43,10 @@ output_format:
   openvino:
     default: true
     input_format: BGR
-compression: null
+optimisations:
+  nncf_quantization:
+    config: "compression_config.json"
```
Comment (evgeny-izutov) [inline pytorch_toolkit/object_detection/model_templates/person-detection/person-detection-0202/template.yaml:48]: ditto

File: pytorch_toolkit/object_detection/model_templates/person-vehicle-bike-detection/person-vehicle-bike-detection-2002/template.yaml [hunk 6]
```diff
@@ -45,7 +45,10 @@ output_format:
   openvino:
     default: true
     input_format: BGR
-compression: null
+optimisations:
+  nncf_quantization:
+    config: "compression_config.json"
```
Comment (evgeny-izutov) [inline pytorch_toolkit/object_detection/model_templates/person-vehicle-bike-detection/person-vehicle-bike-detection-2002/template.yaml:50]: ditto

File: pytorch_toolkit/object_detection/model_templates/vehicle-detection/vehicle-detection-0202/template.yaml [hunk 7]
```diff
@@ -43,7 +43,10 @@ output_format:
   openvino:
     default: true
     input_format: BGR
-compression: null
+optimisations:
+  nncf_quantization:
+    config: "compression_config.json"
```
Comment (evgeny-izutov) [inline pytorch_toolkit/object_detection/model_templates/vehicle-detection/vehicle-detection-0202/template.yaml:48]: ditto

**Review by evgeny-izutov — CHANGES_REQUESTED** [review]

**Comment by evgeny-izutov** [comment]
@LeonidBeynenson

- Fix pylint issue (see http://213.221.44.203/job/ote/job/precommit/job/pt_object_detection/531/)

> [log_009_2020-12-22_04-30-32.zip](https://github.com/openvinotoolkit/training_extensions/files/5727443/log_009_2020-12-22_04-30-32.zip)

- Why didn't new NNCF tests pass on CI (locally only)?

**Comment by LeonidBeynenson** [comment]
run tests

**Comment by LeonidBeynenson** [comment]
run tests

**Comment by LeonidBeynenson** [comment]
run tests

**Review by evgeny-izutov — APPROVED** [review]

File: pytorch_toolkit/object_detection/model_templates/face-detection/face-detection-0202/template.yaml (hunk 1 above)
Comment (LeonidBeynenson) [reply pytorch_toolkit/object_detection/model_templates/face-detection/face-detection-0202/template.yaml]: Fixed

**Pull request merged**